examples/python-guide/logistic_regression.py

```diff
@@ -113,5 +109,5 @@ def experiment(objective, label_type, data):
      for k in range(K)]
 B = [experiment('xentropy', label_type='binary', data=DATA)['time']
      for k in range(K)]
-print('Best `binary` time: ' + str(min(A)))
-print('Best `xentropy` time: ' + str(min(B)))
+print(f"Best `binary` time: {str(min(A))}")
+print(f"Best `xentropy` time: {str(min(B))}")
```

Comment:
```suggestion
print(f"Best `binary` time: {min(A)}")
print(f"Best `xentropy` time: {min(B)}")
```

`str()` is no longer necessary inside an f-string, so I think this can be simplified.